packages/graph/src/index.js

```diff
@@ -15,4 +15,4 @@ limitations under the License.
 export { default as buildGraphData } from './buildGraphData';
 
 export { default as Graph } from './components/Graph';
```

Comment:
Path update here?
```suggestion
export { default as Graph } from './components/legacy/Graph';
```

Reply:
No this is right, `Graph` is the new Carbon Charts based graph component, `PipelineGraph` is the legacy one that was previously dark launched on the run details page.